packages/brand-moment/src/BrandMoment.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactNode, ReactElement } from "react"
+import { Heading, Box, Paragraph } from "@kaizen/component-library"
+import { Button, ButtonProps } from "@kaizen/draft-button"
+import { SceneProps } from "@kaizen/draft-illustration"
+import { assetUrl } from "@kaizen/hosted-assets"
+import classnames from "classnames"
+import styles from "./BrandMoment.scss"
+
+type Props = {
+  backgroundColor: "blue" | "purple" | "green" | "red"
+  header: ReactNode
+  illustration: ReactElement<SceneProps>
+  heading: ReactNode
+  subheading?: ReactNode
+  body?: ReactNode
+  primaryAction: ButtonProps
+  secondaryAction?: ButtonProps
+  footer?: ReactNode
+}
+
+export const BrandMoment = (props: Props) => (
+  <div
+    className={classnames(styles.body, {
+      [styles.blue]: props.backgroundColor === "blue",
+      [styles.green]: props.backgroundColor === "green",
+      [styles.red]: props.backgroundColor === "red",
+    })}
+  >
+    <header className={styles.header}>{props.header}</header>
+    <main className={styles.main}>
```

Comment:
Nice use of semantic HTML 💯

Reply:
Here's to hoping consumers don't put this component inside a `<main>` element 😅